### PR TITLE
Update schema : Caleb

### LINKED ISSUE #170: Fix schema.sql to add distance to body assignment
As a developer, I want to define a database schema that models the key data (weather, celestial bodies, aurora activity, etc.), so the system can store, retrieve, and relate the information in a structured manner, ensuring the stargazing data is accurate and easy to query.

- Update schema.sql to have distance in body assignment
- Update fake data with distances

### PR DESCRIPTION
**Pull request by CalebMathers**
This closes #170 

Added distance_km to body assignment, removed check from aurora alert, changed visibility from smallint to int

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -44,6 +44,7 @@ CREATE TABLE body_assignment (
     at TIMESTAMP NOT NULL,
     azimuth FLOAT NOT NULL,
     altitude FLOAT NOT NULL,
+    distance_km FLOAT NOT NULL,
```
Comment (MurasatoNaoya) [inline database/schema.sql:47]: A good addition, distance is an import factor that dictates whether a body is visible to a given subscriber. Not sure why this wasn't done initially or why I didn't catch this. LGTM.

File: database/schema.sql [hunk 2]
```diff
@@ -77,7 +78,7 @@ CREATE TABLE forecast (
     precipitation_probability_percent SMALLINT NOT NULL,
     precipitation_mm FLOAT NOT NULL,
     cloud_coverage_percent SMALLINT NOT NULL,
-    visibility_m SMALLINT NOT NULL,
+    visibility_m INT NOT NULL,
```
[the remark this answers is not in the document]
Comment (AlexOBarnes) [reply database/schema.sql:81]: Yeah very very large we had problems when inserting

File: database/schema.sql (hunk 1 above)
Comment (AlexOBarnes) [reply database/schema.sql:47]: Good update

File: database/schema.sql [hunk 3]
```diff
@@ -10,7 +10,7 @@ CREATE TABLE aurora_colour (
 
 CREATE TABLE aurora_alert (
     alert_id BIGINT GENERATED ALWAYS AS IDENTITY,
-    alert_time TIMESTAMP NOT NULL CHECK(alert_time<=CURRENT_TIMESTAMP),
+    alert_time TIMESTAMP NOT NULL,
```
Comment (AlexOBarnes) [inline database/schema.sql:13]: Was the constraint causing an issue?

**Review by AlexOBarnes — APPROVED** [review]
Looks great

**Pull request merged**